from src.core.exceptions.base import BaseAppError


class InvalidAuthDataError(BaseAppError):
    pass
